### PR TITLE
Add bezier curve option for connecting lines

### PR DESCRIPTION
**Pull request by FaultyFunctions**
This was just to make the connecting lines look a bit fancier and it works really well with the new grid options. I have set it as the default value but if you wish the straight lines to be the default let me know. Thanks!

### CODE REVIEW

**Comment by blurymind** [comment]
Can you add a screenshot please :) sounds good!

**Comment by FaultyFunctions** [comment]
Sure!
Bezier Curves:
image

Straight Lines:
image

File: src/js/classes/settings.js [hunk 1]
```diff
@@ -141,11 +141,16 @@ export const Settings = function(app) {
...
 
-  // Markup language
+  // Playtest style
   this.playtestStyle = ko
     .observable(storage.getItem('playtestStyle') || 'chat')
     .extend({ persist:'playtestStyle' });
 
+  // Line Style
+  this.lineStyle = ko
+  .observable(storage.getItem('lineStyle') || 'Bezier')
```
Comment (blurymind) [inline src/js/classes/settings.js:151]: can we make straight the default? They read better when you have loads and loads of nodes, its also less intrusive of a change to existing users

File: src/js/classes/workspace.js [hunk 2]
```diff
@@ -448,51 +448,132 @@ export const Workspace = function(app) {
...
+              x1: to.x + normal.x * arrowLength,
+              y1: to.y + normal.y * arrowLength,
+              x2: to.x - normal.x * arrowWidth - normal.y * arrowHeight,
+              y2: to.y - normal.y * arrowWidth + normal.x * arrowHeight,
+              x3: to.x - normal.x * arrowWidth + normal.y * arrowHeight,
+              y3: to.y - normal.y * arrowWidth - normal.x * arrowHeight
+            });
+          }
+        }
+      } else {
```
Comment (blurymind) [inline src/js/classes/workspace.js:499]: why not simply wrap both instead like so
if (node.linkedTo().length) {
    if (app.settings.lineStyle() === "straight") {
       ....
    } else {
       ....
    }
}

**Comment by blurymind** [comment]
Code looks good 👍 again very clean PR. I could only see a couple of minor things :)

File: src/js/classes/workspace.js (hunk 2 above)
Comment (FaultyFunctions) [reply src/js/classes/workspace.js:499]: Good catch! Updating now.

**Comment by FaultyFunctions** [comment]
Okay should be good now!

**Comment by blurymind** [comment]
Thanks for updating this. Will have another look tonight :)

**Comment by blurymind** [comment]
Merged! 🎉 
Thank you for working on this. Looks nice :)

**Pull request merged**